sand/controllers/transform.py

```diff
@@ -30,7 +38,11 @@ def filter_traceback_errors() -> str:
     return "".join(traceback.format_exception(exc, value, tb))
 
 
-def transform_map(transform_func: Callable[[Any, Context], Any], data: Iterable[Tuple[int, List, Context]],
+Item = Any
+ItemIndex = int
+
+
+def transform_map(transform_func: Callable[[Any, Context], Any], data: Tuple[ItemIndex, Item, Context],
```

Comment:
data: Tuple[ItemIndex, Item, Context] where is Iterable?

Reply:
updated with iterable